### PR TITLE
Scaling CTO Talk

### PR DESCRIPTION
**Pull request by OnFreund**

### CODE REVIEW

File: _posts/2016-05-03-scaling-cto-talk.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout:       post
+title:        The Scaling CTO - Talk from the Wheelhouse CTO Summit at the NASDAQ
+author:       on_freund
+summary:
+image:        //res.cloudinary.com/wework/image/upload/c_lfill,f_auto,fl_progressive,g_north,h_1400,w_1400/v1461959537/engineering/scaling_cto_talk.jpg
+categories:   engineering process culture
+wistia_video: wistia_async_sbkrpcark4
+---
+
```
Comment (mattjstar) [inline _posts/2016-05-03-scaling-cto-talk.markdown:10]: You ready to shill???? Can you give more context? Like a tl;dr paragraph with what it's generally about (SEO JUICE!)

Comment (OnFreund) [inline _posts/2016-05-03-scaling-cto-talk.markdown:10]: Not used to shameless self promotion, but I see your point about SEO

File: _posts/2016-05-03-scaling-cto-talk.markdown [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout:       post
+title:        The Scaling CTO - Talk from the Wheelhouse CTO Summit at the NASDAQ
```
Comment (mattjstar) [inline _posts/2016-05-03-scaling-cto-talk.markdown]: Also could consider shilling up the title a bit:

How to scale your engineering organization? How to scale as a CTO etc...

"Talk from the Wheelhouse CTO Summit" can be subheader or H1 at the top of the post.

File: _posts/2016-05-03-scaling-cto-talk.markdown (hunk 1 above)
Comment (mattjstar) [inline _posts/2016-05-03-scaling-cto-talk.markdown:10]: yeah it feels dirty. I leave this comment on pretty much everyone's blog post...

**Comment by OnFreund** [comment]
@mattjstar incorporated your feedback

**Comment by AV4TAr** [comment]
👍

File: _posts/2016-05-03-scaling-cto-talk.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+author:       on_freund
+summary:
+image:        //res.cloudinary.com/wework/image/upload/c_lfill,f_auto,fl_progressive,g_north,h_1400,w_1400/v1461959537/engineering/scaling_cto_talk.jpg
+categories:   engineering process culture
+wistia_video: wistia_async_sbkrpcark4
+---
+
+### The Scaling CTO Talk at the Wheelhouse CTO Summit at the NASDAQ
+
+As VP Engineering at WeWork, I had a great time attending the [Wheelhouse CTO Summit held at the NASDAQ](http://summits.wheelhouse.io/newyork/index.html) a few months ago.
```
Comment (mattjstar) [inline _posts/2016-05-03-scaling-cto-talk.markdown]: I see your point now about the "VP of Engineering" here, I feel like it's self evident by the "CTO Summit" title and you have it in your badge too, so probably not necessary.

**Comment by mattjstar** [comment]
other than the choice about including or not including "VP" in the first paragraph, looks great! 👍

File: _posts/2016-05-03-scaling-cto-talk.markdown [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
 {% include wistia.html %}
 
+
+Some of the topics I cover:
+
+* Companies at different stages require different skills from successful engineering leaders (2:30).
+* Everyone knows that as the company grows you're expected to make the shift from a technical manager to a peoples manager, but what does it mean? (starting at 3:40)
+* What's the difference between speed and agility and why are we moving so slowly? (8:45)
+* What is Conway's law and how can you use it to your advantage? (12:02)
+* What can architectural patterns teach us about product teams organizational structures? (12:37)
```
Comment (GLosch) [inline _posts/2016-05-03-scaling-cto-talk.markdown]: `product teams'` (needs an apostrophe)

File: _posts/2016-05-03-scaling-cto-talk.markdown [hunk 5]
```diff
@@ -1,17 +1,28 @@
...
 
 There were a lot of really great talks by some of the top engineering leaders in NY. Here's one by yours truly:
 
 {% include wistia.html %}
 
+
+Some of the topics I cover:
+
+* Companies at different stages require different skills from successful engineering leaders (2:30).
+* Everyone knows that as the company grows you're expected to make the shift from a technical manager to a peoples manager, but what does it mean? (starting at 3:40)
```
Comment (GLosch) [inline _posts/2016-05-03-scaling-cto-talk.markdown]: `to a people's manager` (needs an apostrophe...I think)

**Comment by dvmlls** [comment]
👎 not about scala as i had originally thought

File: _posts/2016-05-03-scaling-cto-talk.markdown (hunk 4 above)
Comment (OnFreund) [inline _posts/2016-05-03-scaling-cto-talk.markdown]: There's actually a missing "and" in there.

**Pull request merged**